recipes/tritimap/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "0.9.0" %}
+{% set sha256 = "c3a0249f4622fe7b1f4d2d6b716dfb0dd5a9bd6414ac2669ebc3239d50855c58" %}
+
+
+package:
+  name: tritimap
+  version: {{ version }}
+
+source:
+  url: https://github.com/fei0810/Triti-Map/archive/{{ version }}.tar.gz
```

Comment:
https://github.com/fei0810/Triti-Map/archive/{{ version }}.tar.gz should replace to
https://github.com/fei0810/Triti-Map/archive/v{{ version }}.tar.gz